pepper-apis/dsm-core/src/main/resources/liquibase/CMI/PEPPER-445_tissueReceivedEvent.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<databaseChangeLog
-        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
-
-    <changeSet author="simone" id="PEPPER-445_samplesReceivedEvent">
-        <insert tableName="event_type">
-            <column name="ddp_instance_id"
-                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE study_guid='cmi-lms')"></column>
-            <column name="event_name" value="REQUIRED_SAMPLES_RECEIVED"/>
-            <column name="event_description" value="Samples received"/>
-            <column name="event_type" value="RECEIVED"/>
-        </insert>
-        <insert tableName="event_type">
-            <column name="ddp_instance_id"
-                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='osteo2')"></column>
-            <column name="event_name" value="REQUIRED_SAMPLES_RECEIVED"/>
-            <column name="event_description" value="Samples received"/>
-            <column name="event_type" value="RECEIVED"/>
-        </insert>
-    </changeSet>
-
-</databaseChangeLog>
```

Comment:
if we have to run it again at some point maybe we can keep this with a new id?

Reply:
then we have to do it per release. if we do it per just db sql we don't need to do it pr RC…